pyproject.toml

```diff
@@ -1,13 +1,15 @@
-# pyproject.toml
 [build-system]
 requires = [
-    "setuptools>=42",
-    "wheel",
-    "setuptools_scm[toml]>=3.4"
+  "setuptools >= 42",
+  "wheel",
+  "setuptools_scm[toml]>=3.4"
```

Comment:
Is there supposed to be a space around `>=` or not?

Reply:
it appears to not matter... but seems like I should at least be consistent! 😂 
i removed the space